client/src/App.js

```diff
@@ -36,8 +46,40 @@ class App extends React.Component {
                         <ErrorHandler>
                             <Route path="/map" component={MapPage}/>
                             <Route exact path="/">
-                                <div style={{display: 'flex', justifyContent: 'center'}}>
-                                    <h1>Welcome!</h1>
+                                <div id="welcome-page-body">
+                                    <h1>
+                                        Welcome to Monuments + Memorials
+                                    </h1>
+                                    <div>
+                                        <img className='homePageImage' src={process.env.PUBLIC_URL + '/home_page_image.jpg'}/>
```

Comment:
`<img>` elements should always have an `alt` prop so that if for some reason they can't be displayed, something else is.

In this case, I think it can just be an empty string, but we should still have it.

`<img className='homePageImage' src={process.env.PUBLIC_URL + '/home_page_image.jpg'} alt=''/>`

Reply:
This should really be done using a `div` with a `background: url('...')` so that we can resize and fit as needed. Using `img` means that the image will need to be specific dimensions which is very limiting